app/namespaces/test_adjustment/test_adjustment_service.py

```diff
@@ -178,12 +178,12 @@ def get_adjusted_estimate(self, test_adj, ind_se, ind_sp, ind_se_n, ind_sp_n,
         if pd.isna(test_adj):
 
             # Independent evaluation is available
```

Comment:
Change the prioritization order so Author evaluation is first

Reply:
[e424cf4](https://github.com/serotracker/iit-backend/pull/327/commits/e424cf4385419691d1e89338aa43680dffc0d1b2)